test/development/watch-config-file/index.test.ts

```diff
@@ -22,27 +22,37 @@ describe('watch-config-file', () => {
         `,
       },
       dependencies: {},
-      skipStart: true,
     })
   })
   afterAll(() => next.destroy())
 
   it('should output config file change', async () => {
-    // next dev test-dir
-    await next.start(true)
```

Comment:
When I actually tried accessing the server it failed with weird `useContext` errors.

I'm not sure what it does but it works now.